src/Tizen.System.Information/Usage/ProcessMemoryUsage.cs

```diff
@@ -209,6 +304,30 @@ public void Update(IEnumerable<int> pid)
                 Usages[i] = Marshal.PtrToStructure<Interop.RuntimeInfo.ProcessMemoryInfo>(ptr);
                 ptr += Marshal.SizeOf<Interop.RuntimeInfo.ProcessCpuUsage>();
             }
+
+            try
+            {
+                GetProcessMemoryValueInt(Interop.RuntimeInfo.ProcessMemoryInfoKey.Gpu, out Gpus);
+            }
+            catch (NotSupportedException)
+            {
```

Comment:
Print any warning logs?

Reply:
Log will printed by GetProcessMemoryValueInt function. 
So just skip in here.

Reply:
How about skipping this exception throwing in `GetProcessMenoryValueInt` method?